ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/workers-rhel8/provision/cucushift-installer-rehearse-azure-ipi-workers-rhel8-provision-chain.yaml

```diff
@@ -16,7 +16,7 @@ chain:
   - name: RHEL_WORKER_COUNT
     default: 3
   - name: RHEL_VM_SIZE
-    default: "Standard_D4s_v3"
```

Comment:
So this size does not support `accelerated-networking`, right? If yes, I guess `cucushift-installer-rehearse-azure-ipi-proxy-workers-rhcos-rhel8-provision` also need to update

Reply:
`Standard_D4s_v3` supports `accelerated-networking`, but scaleup still failed if only enabling accelerated-networking or only changing size to `Standard_D8s_v3`, need to update both to make it work.

Since issue only exists when scaling up 3 or more rhel worker nodes on 4.16, so only update `cucushift-installer-rehearse-azure-ipi-workers-rhel8`.

Reply:
good to know.